### PR TITLE
Add support for Pensando DSC card

### PR DESCRIPTION
**Pull request by vrindle**

### CODE REVIEW

File: deployment/sriov-network-operator/templates/configmap.yaml [hunk 1]
```diff
@@ -23,3 +23,5 @@ data:
   Broadcom_bnxt_BCM75508_2x100G: "14e4 1750 1806"
   Qlogic_qede_QL45000_50G: "1077 1654 1664"
   Red_Hat_Virtio_network_device: "1af4 1000 1000"
+  Pensando_ionic_DSC: "1dd8 1002 1003"
+
```
Comment (SchSeba) [inline deployment/sriov-network-operator/templates/configmap.yaml:27]: please remove this empty line

**Comment by adrianchiris** [comment]
Hi @vrindle !

according to: https://github.com/k8snetworkplumbingwg/sriov-network-operator/blob/master/doc/supported-hardware.md

you should create an issue requesting to add support for this NIC, with information of what was tested for this initial support.
In addition, who will be the contact person to reach out in case there are issues raised by users regarding this NIC ?

ATM we dont have any pensando folks involved in the project AFAIK

Thanks!

**Comment by SchSeba** [comment]
Hi @vrindle any update on this PR?

**Comment by jjuele** [comment]
Can one of the admins verify this patch?

**Comment by vrindle** [comment]
Pensando is fixing this issue in driver+fw. This pull request can be closed

**Pull request closed without merging**